source/agora/consensus/EnrollmentManager.d

```diff
@@ -167,6 +167,46 @@ public class EnrollmentManager
         ***********************************************************************/
 
         private PreImageCache preimages;
+
+        /***********************************************************************
+
+            Consume the current cycle
+
+            This will first populate the caches (seeds and preimages)
+            as necessary, then increase the `index` by one, or reset it to 0
+            and increase `nonce` if necessary.
+            Note that the increment is done after the caches are populated,
+            so `consume` needs to be called once this node has confirmed it
+            is part of consensus.
+
```

Comment:
Need to add documentation for the parameter.

Reply:
Good catch, fixed